docs/proxy/datacenter_proxy/examples.md

```diff
@@ -0,0 +1,700 @@
+        groups: ["SHADER", "BUYPROXIES94952"],
+    });
+
+    const crawler = new Apify.PuppeteerCrawler({
+        requestList,
+        proxyConfiguration,
+        handlePageFunction: async ({ page, request, proxyInfo }) => {
+            return Apify.pushData({ title: await page.title() });
+        },
+        handleFailedRequestFunction: ({ request }) => {
+            console.error("Request failed", request.url, request.errorMessages);
+        },
+    });
+
+    await crawler.run();
+});
+</marked-tab>
+
+
+<marked-tab header="launchPuppeteer()" lang="javascript">
+const Apify = require("apify");
+
+Apify.main(async () => {
+    const proxyConfiguration = await Apify.createProxyConfiguration({
+        groups: ["SHADER", "BUYPROXIES94952"],
+    });
+
+    const browser = await Apify.launchPuppeteer({
+        proxyUrl: proxyConfiguration.newUrl("my_session"),
+    });
+
+    const page = await browser.newPage();
+
+    await page.goto("http://www.example.com");
+
+    const html = await page.content();
+
+    console.log("HTML:");
+    console.log(html);
+});
+</marked-tab>
+
+
+<marked-tab header="requestAsBrowser()" lang="javascript">
+const Apify = require("apify");
+
+Apify.main(async () => {
+    const proxyConfiguration = await Apify.createProxyConfiguration({
+        groups: ["SHADER", "BUYPROXIES94952"]
+    });
+    const proxyUrl = proxyConfiguration.newUrl();
+    const url = "https://api.apify.com/v2/browser-info";
+
+    try {
+        const response1 = await Apify.utils.requestAsBrowser({
+            url,
+            proxyUrl,
+            json: true
+        });
+        const response2 = await Apify.utils.requestAsBrowser({
+            url,
+            proxyUrl,
+            json: true
+        });
+        console.log(response1.body.clientIp);
+        console.log("should be different than");
+        console.log(response2.body.clientIp);
+    } catch (e) {
+        console.error(e);
+    }
+});
+</marked-tab>
+```
+
+
+### [](#single-ip-address-from-a-group) Single IP address from a group
+
+Using [PuppeteerCrawler](https://sdk.apify.com/docs/api/puppeteer-crawler#docsNav), get a new IP address selected from the `SHADER` proxy group for each browser opened during an entire run.
+
+If the IP fails, all browsers using it are retired and new ones, each with its own new random IP, are started.
+
+```marked-tabs
+<marked-tab header="PuppeteerCrawler" lang="javascript">
+const Apify = require("apify");
+
+Apify.main(async () => {
+    const requestList = await Apify.openRequestList(
+        "my-list", ["http://www.example.com"]
+    );
+    const proxyConfiguration = await Apify.createProxyConfiguration({
+        // if you need to use more then one group
+        // simply add the additional ID to the array below
+        groups: ["SHADER"],
+    });
+
+    const crawler = new Apify.PuppeteerCrawler({
+        requestList,
+        proxyConfiguration,
+        useSessionPool: true,
```

Comment:
This will use Random IPs, not a single IP as the title suggests. I see there's an explanation in the paragraph below the title, but I still find it confusing. Single IP to me means static IP. The same IP for all requests.